core/errors/namer.h

```diff
@@ -20,7 +20,6 @@ constexpr ErrorClass DynamicConstant{4014, StrictLevel::False};
 constexpr ErrorClass InvalidClassOwner{4015, StrictLevel::False};
 constexpr ErrorClass RootTypeMember{4016, StrictLevel::False};
 constexpr ErrorClass DynamicConstantAssignment{4017, StrictLevel::False};
-constexpr ErrorClass RepeatedArgument{4018, StrictLevel::False};
```

Comment:
please, to remove errors, comment them out. This allows us to not reuse error codes.